solana/spl_initialized_account/src/pb/mod.rs

```diff
@@ -0,0 +1,82 @@
+// @generated
```

Comment:
There is a bunch of exclusion missing here, a lot of the generated Protobuf in there are not used at all.